src/views/Pipeline.vue

```diff
@@ -117,6 +117,47 @@ export default defineComponent({
         event.target.complete();
       })
     },
+    async skipJob (job: any) {
+      const alert = await alertController
+        .create({
+          header: this.$t('Skip job'),
+          message: this.$t('Skipping will run this job at the next occurance based on the temporal expression.'),
+          buttons: [
+            {
+              text: this.$t('Dont skip'),
+              role: 'cancel',
+            },
+            {
+              text: this.$t('Skip'),
+              handler: () => {
+                let skipTime = {};
+                if(this.temporalExpr(job.tempExprId).integer1 === 12){
```

Comment:
`this.temporalExpr(job.tempExprId).integer1` and `this.temporalExpr(job.tempExprId).integer2` can also be stored in a variable and then used everywhere, WDYT?

Reply:
Made the requested change.